dashboard/app/controllers/hoc_download_controller.rb

```diff
@@ -0,0 +1,42 @@
+class HocDownloadController < ApplicationController
+
+  DOWNLOAD_URL_BASE = '//downloads.code.org/hourofcode/'
+
+  def index
+    @product_name = params[:product]
+
+    case @product_name
+      when Script::MINECRAFT_NAME
```

Comment:
Same comment as below, though less important if we stick with two tutorials for forseeable future.. if we add additional tutorials we'll end up with duplicated code.